build/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# If we are using compose.yml use the repo config.toml
+if [ -f /podman_compose/kai/config.toml ]; then
+  cp /podman_compose/kai/config.toml /kai/kai/config.toml
+  sed -i 's/^host\ =.*/host = "kai_db"/g' /kai/kai/config.toml
+  sed -i "s/^database =.*/database = \"$POSTGRES_DB\"/g" /kai/kai/config.toml
+  sed -i "s/^user =.*/user = \"$POSTGRES_USER\"/g" /kai/kai/config.toml
+  sed -i "s/^password =.*/password =\"$POSTGRES_PASSWORD\"/g" /kai/kai/config.toml
+fi
+
+TABLE=applications
+SQL_EXISTS=$(printf '\dt "%s"' "$TABLE")
+if [[ $(PGPASSWORD="$POSTGRES_PASSWORD" psql -h "kai_db" -U $POSTGRES_USER -d $POSTGRES_DB -c "$SQL_EXISTS") ]]
+then
+    echo "load-data has run already"
+else
+    echo "load-data has not run yet, starting ..."
```

Comment:
Should we put a flag to skip load-data in case we want to pull from conveyor instead? Or we could do it in a follow up if we're just trying to containerize the standard demo real quick

Reply:
Makes sense. I can set add an env var and add a conditional.